data/strings/strings.txt

```diff
@@ -10021,6 +10021,10 @@
     zh-Hans = 示例值
     zh-Hant = 範例值
 
+  [editor_invalid_format]
```

Comment:
What is wrong with the existing translation editor_correct_mistake ? Why it can't be reused here?

Reply:
`editor_correct_mistake` is not descriptive of what's wrong & it's being used in a few other instances in the codebase, so it cannot be removed.

Reply:
"Format not valid" is even worse: not every non-developer user can realize what "format" is. Considering that the error message should appear immediately after a typed symbol, it should be clear to a user what went wrong. And we're talking about the "Advanced" mode, right?

Reply:
`Correct mistake` only signifies user that there is a mistake.
`Format is not valid!` lets, users know that the mistake they are making is the wrong format.
This is the reason I went on with changing the string in this instance.
This error will only be displayed in advanced mode.